### PR TITLE
util.plot.make_map: Add figsize argument 

### PR DESCRIPTION
**Pull request by chahank**
This addresses issue #135 and adds the keyword argument `figsize=(9, 13)` to `util.plot.make_plot` and to all the function/methods that call it too.

### CODE REVIEW

File: climada/entity/exposures/base.py [hunk 1]
```diff
@@ -399,6 +400,7 @@ def plot_scatter(self, mask=None, ignore_zero=False, pop_name=True,
             extend (str, optional): extend border colorbar with arrows.
                 [ 'neither' | 'both' | 'min' | 'max' ]
             axis (matplotlib.axes._subplots.AxesSubplot, optional): axis to use
+            figsize (tuple): figure size for plt.subplots
```
Comment (tovogt) [inline climada/entity/exposures/base.py:403]:
```suggestion
            figsize (tuple, optional): figure size for plt.subplots
```

File: climada/hazard/base.py [hunk 2]
```diff
@@ -665,14 +665,15 @@ def local_exceedance_inten(self, return_periods=(25, 50, 100, 250)):
...
-                          smooth=True, axis=None, **kwargs):
+                          smooth=True, axis=None, figsize=(9, 13), **kwargs):
         """Compute and plot hazard exceedance intensity maps for different
         return periods. Calls local_exceedance_inten.
 
         Parameters:
             return_periods (tuple(int), optional): return periods to consider
             smooth (bool, optional): smooth plot to plot.RESOLUTIONxplot.RESOLUTION
             axis (matplotlib.axes._subplots.AxesSubplot, optional): axis to use
+            figsize (tuple): figure size for plt.subplots
```
Comment (tovogt) [inline climada/hazard/base.py:676]:
```suggestion
            figsize (tuple, optional): figure size for plt.subplots
```

File: climada/hazard/base.py [hunk 3]
```diff
@@ -1107,6 +1109,7 @@ def _event_plot(self, event_id, mat_var, col_name, smooth, axis=None, **kwargs):
             col_name (sparse matrix): Colorbar label
             smooth (bool, optional): smooth plot to plot.RESOLUTIONxplot.RESOLUTION
             axis (matplotlib.axes._subplots.AxesSubplot, optional): axis to use
+            figsize (tuple): figure size for plt.subplots
```
Comment (tovogt) [inline climada/hazard/base.py:1112]:
```suggestion
            figsize (tuple, optional): figure size for plt.subplots
```

File: climada/hazard/trop_cyclone.py [hunk 4]
```diff
@@ -225,6 +225,7 @@ def video_intensity(track_name, tracks, centroids, file_name=None,
             file_name (str, optional): file name to save video, if provided
             writer = (matplotlib.animation.*, optional): video writer. Default:
                 pillow with bitrate=500
+            figsize (tuple): figure size for plt.subplots
```
Comment (tovogt) [inline climada/hazard/trop_cyclone.py:228]:
```suggestion
            figsize (tuple, optional): figure size for plt.subplots
```

File: climada/util/plot.py [hunk 5]
```diff
@@ -76,6 +77,7 @@ def geo_bin_from_array(array_sub, geo_coord, var_name, title, pop_name=True,
         extend (str, optional): extend border colorbar with arrows.
             [ 'neither' | 'both' | 'min' | 'max' ]
         proj (ccrs): coordinate reference system of the given data
+        figsize (tuple): figure size for plt.subplots
```
Comment (tovogt) [inline climada/util/plot.py:80]:
```suggestion
        figsize (tuple, optional): figure size for plt.subplots
```

File: climada/util/plot.py [hunk 6]
```diff
@@ -161,6 +164,7 @@ def geo_scatter_from_array(array_sub, geo_coord, var_name, title,
         extend (str, optional): extend border colorbar with arrows.
             [ 'neither' | 'both' | 'min' | 'max' ]
         proj (ccrs): coordinate reference system used in coordinates
+        figsize (tuple): figure size for plt.subplots
```
Comment (tovogt) [inline climada/util/plot.py:167]:
```suggestion
        figsize (tuple, optional): figure size for plt.subplots
```

File: climada/util/plot.py [hunk 7]
```diff
@@ -225,6 +230,7 @@ def geo_im_from_array(array_sub, coord, var_name, title,
         proj (ccrs): coordinate reference system used in coordinates
         smooth (bool, optional): smooth plot to RESOLUTIONxRESOLUTION. Default:
             True.
+        figsize (tuple): figure size for plt.subplots
```
Comment (tovogt) [inline climada/util/plot.py:233]:
```suggestion
        figsize (tuple, optional): figure size for plt.subplots
```

**Review by tovogt — APPROVED** [review]
Apart from some `optional` notes in the docstrings, this is fine from my side. If you feel generous, please replace all the docstrings you touched by NumPy style docstrings ;)

File: climada/entity/exposures/base.py [hunk 8]
```diff
@@ -456,11 +461,12 @@ def plot_hexbin(self, mask=None, ignore_zero=False, pop_name=True,
                           self.gdf.longitude[mask][pos_vals].values], axis=1)
         return u_plot.geo_bin_from_array(value, coord, cbar_label, title,
                                          pop_name, buffer, extend, proj=crs_epsg,
-                                         axes=axis, **kwargs)
+                                         axes=axis, figsize=fisgize, **kwargs)
```
Comment (emanuel-schmid) [inline climada/entity/exposures/base.py:464]:
```suggestion
                                         axes=axis, figsize=figsize, **kwargs)
```

**Review by emanuel-schmid — CHANGES_REQUESTED** [review]

**Review by emanuel-schmid — APPROVED** [review]

**Pull request merged**